assets/blocks/course-outline/outline-block/existing-lessons-modal/lessons-modal-style.scss

```diff
@@ -0,0 +1,102 @@
+.wp-block-sensei-lms-course-outline__existing-lessons-modal {
+	min-width: 566px;
+	max-width: 697px;
+
+	/* Reset some components styles */
+	.components-base-control__field,
+	.components-checkbox-control__input-container,
+	ul,
+	li {
+		margin: 0;
+	}
+
+	&__notice {
+		margin: 0 0 15px;
+	}
+
+	&__existing-lessons {
+		overflow: auto;
+		height: calc(100vh - 225px);
```

Comment:
With scroll, it's not with a good behavior. I imagine it would be nice to keep the filters and actions fixed and scroll only in the list. WDYT?

https://github.com/Automattic/sensei/assets/876340/d0f020f6-2408-495b-935d-becf1ab10de9

Sorry if I missed something, but to not create a lot of orphan lessons, I added the `tr` directly through the HTML. But I think it would be the behavior of having many lessons, right?

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/7368/commits/9e1d6ed78fafd4d2e4e94b51bfd935e3d66fa3d5

Reply:
Thanks! It's much better already, but in some window heights, it's still with an unexpected scroll.

https://github.com/Automattic/sensei/assets/876340/87469a7f-d850-4bfa-bba9-a967b4c4b929

Reply:
Thanks @renatho!
Fixed it here: https://github.com/Automattic/sensei/pull/7368/commits/cdc1d97b3cd59bd391ea952662a0b8ccf53c1809